ext/date/tests/test-parse-from-format.phpt

```diff
@@ -45,7 +45,18 @@ object(DateTime)#1 (3) {
 
 string(13) "Y-m-d\TH:i:sO"
 string(24) "2008-07-08T22:14:12+0200"
-object(DateTime)#3 (3) {
+object(DateTime)#%d (3) {
+  ["date"]=>
+  string(26) "2008-07-08 22:14:12.000000"
+  ["timezone_type"]=>
+  int(1)
+  ["timezone"]=>
+  string(6) "+02:00"
```

Comment:
The test expectation needs to be [fixed](https://ci.appveyor.com/project/php/php-src/builds/43182222/job/2f59vn7nvu6mmvoc?fullLog=true#L6800):
```suggestion
  ["timezone_type"]=>
  int(3)
  ["timezone"]=>
  string(11) "Europe/Oslo"
```

Reply:
Thanks. I couldn't figure out why this result showed a different timezone type.

Reply:
That is because there is no explicit timezone (offset) in `$formatted` for ISO9075 format.